docs/examples/safe-remote.rst

```diff
@@ -6,18 +6,18 @@ Safe Remote Purchase
 
 Purchasing goods remotely currently requires multiple parties that need to trust each other.
 The simplest configuration involves a seller and a buyer. The buyer would like to receive
-an item from the seller and the seller would like to get money (or an equivalent)
+an item from the seller and the seller would like to get currency
```

Comment:
Hm... I'm not sure "currency" is that much better than "money"... maybe we should go for "some compensation, e.g. Ether," in the first occurrence here and then move to "Ether" in the further occurrences below (those refer to the example in particular, in which it *is* Ether)...

Reply:
Makes sense, just committed this change.

Reply:
Can you also change the other occurrences of "currency"?

Reply:
Shall I change all instances of currency outwith the changes I made as well?

Reply:
done.